### PR TITLE
Ft bubble capturing

### PR DESCRIPTION
**Pull request by FlorisXll**
Added the feature of bubble capturing.

Also pushed some configurations.

### CODE REVIEW

**Comment by jessetilro** [comment]
When I have encapsulated all enemies, but have not killed them yet, I already advance to the next level. Could you fix this?

**Comment by FlorisXll** [comment]
On it.

File: src/main/java/nl/tudelft/scrumbledore/CollisionsLevelModifier.java [hunk 1]
```diff
@@ -268,21 +281,22 @@ public void detectPlayerBubble(Level level, double delta) {
...
-            Fruit newFruit = new Fruit(enemies.get(i).getPosition().clone(), new Vector(
-                Constants.BLOCKSIZE, Constants.BLOCKSIZE));
-            // Adding a new Fruit element in place of where the enemy died.
+              && enemies.get(i).inBoxRangeOf(bubbles.get(j), Constants.COLLISION_RADIUS)
+              && new Collision(bubbles.get(j), enemies.get(i), delta).colliding()) {            
+            
+            // The enemy gets removed and a new encapsulated enemy will appear.
             enemies.remove(i);
-            fruits.add(newFruit);
+            bubbles.get(j).setHasNPC(true);
```
Comment (jessetilro) [inline src/main/java/nl/tudelft/scrumbledore/CollisionsLevelModifier.java]: I think you should check whether this Bubble does not already have an NPC, currently this implementation allows for a Bubble to encapsulated multiple enemies.

**Comment by FlorisXll** [comment]
Resolved all comments.

**Comment by jessetilro** [comment]
Tip: keep the summary of your commit short and below the character limit, put the details in the description. (Summary en description are separated by two newlines.)

**Comment by jessetilro** [comment]
Another bug I detected during manual testing: when the player jumps on top of a bubble containing an enemy, it bounces instead of killing the enemy. Could you fix collision handling so that the player only bounces on bubbles not containing an enemy?
<bug

**Comment by FlorisXll** [comment]
Couldn't really call it a fix, because it is just feature that you are able to bounce upon a bubble with an enemy in it. We can indeed make the choice of killing the enemy when bouncing on its bubble.

**Comment by FlorisXll** [comment]
Resolved all comments.

**Pull request merged**